File: Belajar_Day_1/day1_8.java
import java.util.Scanner;
import java.io.File;
import java.io.FileNotFoundException;
import java.io.FileWriter;
import java.io.IOException;

public class day1_8 {
    public static void main(String[] args) {
        try {
        // //buka file
        // File file = new File("test2.txt");
        // //baca file
        // Scanner reader = new Scanner(file);
        // //tampilkan
        // while (reader.hasNextLine()){
        //     String line = reader.nextLine();
        //     System.out.println(line);
        // } 

        FileWriter writer = new FileWriter("test2.txt",true);
        //tulis
        writer.write("Haloo....\n");
        writer.write("Haloo jugaa\n");
        //tutup file
        //reader.close();
        writer.close();
    // } catch (FileNotFoundException e){
    //     System.out.println("File tidak ditemukan!");
    //     }
        } catch (IOException e){
        System.out.println("File tidak ditemukan!");
        }
    }
}

File: Belajar_Day_1/day1_8.py
# #buka file
# file = open("test.txt","r")

# #baca isi file
# content = file.read()

# #tampilkan isi file
# print(content)

# #tutup file
# file.close()

#cara yang lebih singkat
with open("test.txt", "r") as file:
    content = file.read()
    print(content)

#buat file
with open ("test2.txt", "w") as file:
    file.write("Halo, apa kabar?\n")
    file.write("Test 123\n")
print("File berhasil dibuat")

#append file / menambahkan
with open ("test2.txt", "a") as file:
    file.write("Halo, apa kabar??\n")
    file.write("Test 1234\n")
print("File berhasil dibuat")